lxd/networks.go

```diff
@@ -1845,6 +1846,26 @@ func networkStateGet(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(api.StatusErrorf(http.StatusNotFound, "Network not found"))
 	}
 
+	// If network is of type physical managed, then return info on parent.
+	if n != nil && n.Type() == "physical" && n.IsManaged() {
+		config := n.Config()
+		state, err := resources.GetNetworkState(network.GetHostDevice(config["parent"], config["vlan"]))
+		if err != nil {
+			// If parent is not found, then return response indicating the parent is unavailable.
+			statusErr := api.StatusError{}
+			if errors.As(err, &statusErr) && statusErr.Status() == 404 {
```

Comment:
Please use `api.StatusErrorCheck(err, http.StatusNotFound)` here rather than the manual errors.As call.

Reply:
Fixed, thanks!